code/drasil-gen/lib/Language/Drasil/Generate.hs

```diff
@@ -70,7 +85,8 @@ prntDoc' dt dt' fn format body' sm = do
   where getExt TeX  = ".tex"
         getExt HTML = ".html"
         getExt JSON = ".ipynb"
-        getExt _    = error "We can only write in TeX, HTML and Jupyter Notebook (for now)."
+        getExt Markdown = ".md"
+        getExt _    = error "We can only write in TeX, HTML, Jupyter Notebook, Markdown, and mdBook (for now)."
```

Comment:
A comment here saying that the `MDBook` case is handled above is probably a good idea, for those who don't see it in the previous pattern matching and think there's a mistake (couldn't be me…)

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3823/commits/f7ba8fda90132e900e5349385606160f9c09ac72.